config/config.go

```diff
@@ -82,6 +82,13 @@ type Extensions struct {
 // executing signer actions specific for TBTC application.
 type TBTC struct {
 	TBTCSystem string
+	BTCRefunds BTCRefunds
+}
+
+// BTCRefunds stores configuration related to recovering BTC from a liquidated
+// keep
```

Comment:
Let's just say “a closed keep”, since this will probably become broader in applicability.

Also---periods are nice :grin: